source/renderer/app/stores/SidebarStore.js

```diff
@@ -83,7 +83,7 @@ export default class SidebarStore extends Store {
       [categories.PAPER_WALLET_CREATE_CERTIFICATE.name]: false,
       [categories.STAKING_DELEGATION_COUNTDOWN.name]: false,
       [categories.STAKING.name]: true,
-      [categories.REDEEM_ITN_REWARDS.name]: true,
+      [categories.REDEEM_ITN_REWARDS.name]: false,
```

Comment:
@yakovkaravelov please remove this line completely and clean up any leftovers (e.g. categories).